cluster/validate-cluster.sh

```diff
@@ -52,7 +52,12 @@ CLUSTER_READY_ADDITIONAL_TIME_SECONDS="${CLUSTER_READY_ADDITIONAL_TIME_SECONDS:-
 
 EXPECTED_NUM_NODES="${NUM_NODES}"
 if [[ "${REGISTER_MASTER_KUBELET:-}" == "true" ]]; then
-  EXPECTED_NUM_NODES=$((EXPECTED_NUM_NODES+1))
+  if [[ "${KUBE_EXPERIMENTAL_REPLICATE_EXISTING_MASTER:-}" == "true" ]]; then
+    NUM_MASTERS=$(get-master-replicas-count)
```

Comment:
What do you mean by "we can make assumption". How someone would know to implement this? Currently it's only required to implement some high level methods like kube-up etc. We'd have to make it part of explicit API, which most likely we don't want to do...